src/modals/SendModal/TransactionForm.tsx

```diff
@@ -156,6 +164,25 @@ const SendModalTransactionForm = ({ data, onSubmit, onCancel }: TransactionFormP
           error={gasPriceError}
         />
       </ExpandableSectionStyled>
+      <ExpandableSectionStyled sectionTitleClosed="Script">
+        <Input id="state" placeholder="Code" value={script} onChange={(e) => setScript(e.target.value)} />
+      </ExpandableSectionStyled>
+      <ExpandableSectionStyled sectionTitleClosed="Contract">
+        <Input id="code" placeholder="Code" value={contractCode} onChange={(e) => setContractCode(e.target.value)} />
+        <Input
+          id="state"
+          placeholder="Initial state"
+          value={contractState}
+          onChange={(e) => setContractState(e.target.value)}
+        />
+        <Input
+          id="issue-token-amount"
+          placeholder="Tokens to issue"
+          value={issueTokenAmount}
+          type="number"
+          onChange={(e) => setIssueTokenAmount(e.target.value)}
+        />
```

Comment:
Just to understand this properly, does every call to smart contract going to issue new tokens? :/

Reply:
Nope - you can specify 0 if you want, but yes it happens specifically for contracts.